sdks/python/apache_beam/dataframe/io.py

```diff
@@ -58,6 +58,28 @@
 _DEFAULT_BYTES_CHUNKSIZE = 1 << 20
 
 
+def read_gbq(
+    table, dataset=None, project_id=None, use_bqstorage_api=False, **kwargs):
+  """This function reads data from a BigQuery source and outputs it into
+  a Beam deferred dataframe
+  (https://beam.apache.org/documentation/dsls/dataframes/overview/)
+  Please specify a table in the format 'PROJECT:dataset.table'
+  or use the table, dataset, and project_id args
+  to specify the table. If you would like to utilize the BigQuery
+  Storage API in ReadFromBigQuery,
+    please set use_bq_storage_api to True.
+    Otherwise, please set the flag to false or
+    leave it unspecified."""
```

Comment:
nit: Could you format this with "Args:" instead?
```suggestion
  Args:
    table: ...
    dataset: ...
    project_id: ...
    use_bqstorage_api: ...
```